src/index.html

```diff
@@ -8,6 +8,77 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
```

Comment:
```suggestion
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
```

You need to add empty line between multiple sibling blocks and not between single tags.

Example:

```
<div>
  <p>some text</p>
</div>

<div>
  <p>some text...</p>
</div>
```